src/tests/profiler/unittest/getappdomainstaticaddress.cs

```diff
@@ -47,8 +44,9 @@ private static void LoadCollectibleAssembly()
             object instance = Activator.CreateInstance(testType);
 
             Console.WriteLine(instance.GetHashCode());
-
+            Thread.Sleep(TimeSpan.FromSeconds(2));
             collectibleContext.Unload();
+            Thread.Sleep(TimeSpan.FromSeconds(1));
```

Comment:
Is there any way to achieve this without sleeps?  Invariably when we use sleeps for coordination in tests, eventually it'll fail spuriously.